mbedtls/src/error.rs

```diff
@@ -31,56 +31,98 @@ pub const ERR_UTF8_INVALID: c_int = -0x10000;
 
 macro_rules! error_enum {
     {enum $n:ident {$($rust:ident = $c:ident,)*}} => {
```

Comment:
```suggestion
    {
        const MASK: c_int = $mask:literal;
        enum $n:ident {$($rust:ident = $c:ident,)*}
    } => {
```
See comments below for how this would be used.